sections/email-signup-banner.liquid

```diff
@@ -320,26 +320,6 @@
           "id": "text",
           "default": "<p>Be the first to know when we launch.</p>",
           "label": "t:sections.email-signup-banner.blocks.paragraph.settings.paragraph.label"
-        },
-        {
```

Comment:
For this. I realize now it might now have been clear. I meant specifically the uppercase option. I would touch base with @wiktoriaswiecicka  to confirm

Reply:
We will just be removing uppercase. Updated